metricbeat/docs/modules/http.asciidoc

```diff
@@ -5,8 +5,8 @@ This file is generated! See scripts/docs_collector.py
 [[metricbeat-module-http]]
 == http Module
 
-Http module is a [Metricbeat](https://www.elastic.co/products/beats/metricbeat) used to call arbitrary HTTP endpoints for which not a dedicated metricbeat is available.
-Multiple endpoints can be configured which are polled in a regular interval and the result is shipped to the configured output channel.
+Http module is a [Metricbeat](https://www.elastic.co/products/beats/metricbeat) used to call arbitrary HTTP endpoints for which not a dedicated metricbeat module is available.
```

Comment:
changed to `module used to call arbitrary HTTP endpoints for which a dedicated metricbeat module is not available.`